### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by PrantoshB**
### Hello there 👋

In this branch, I and my coding partner @tanveerisonline have:

- Created a file named schema_based_on_diagram.sql where we have implemented the database from the diagram below: 

image

- Joined tables from many-to-many relationships might not appear in the diagram, but you still need them.
- Add foreign key indexes

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    status varchar(255) not null,
+    patient_id int not null references patients(id),
+)
+
+create table treatments (
+    id int not null references medical_histories(id) primary key,
+    type varchar(255) not null,
+    name varchar(255) not null,
+)
+
```
Comment (TadesseDev) [inline schema_based_on_diagram.sql:19]: - [ ] Dear @PrantoshB, Good job creating the table `medical_histories` and `treatments`. But the relationship between those two tables is a many-to-many. And all "join tables" for the many-to-many relationships are expected to be created, even if they are not shown on the diagram. Please also notice we need a separate `primary key` and `foreign key`.
- [ ] Once you create the association with primary and foreign keys in your new tables don't forget to add/create the `index` for the foreign keys.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7


## Highlights
 - [ ]  Correct GitFlow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (PrantoshB) [reply schema_based_on_diagram.sql:19]: Thanks @TadesseDev , I've added the table and re-submitted.

**Review by PinkMoon25 — APPROVED** [review]
Hi @PrantoshB @tanveerisonline , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
gitflow :heavy_check_mark: 
pair-programming :heavy_check_mark: 
good PR and commits :heavy_check_mark: 
Join table :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PrantoshB** [comment]
Thanks @PinkMoon25 for reviewing our PR. As it meets the requirements, I'm merging it.

**Pull request merged**